tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -641,7 +646,9 @@ def test_user_in_team_without_access_cannot_delete_dashboard(self):
         assert response.status_code == 403
 
 
-class OrganizationDashboardDetailsPutTest(OrganizationDashboardDetailsTestCase):
+class OrganizationDashboardDetailsPutTest(
+    OrganizationDashboardDetailsTestCase, MetricsEnhancedPerformanceTestCase
```

Comment:
TODO: Can I avoid pulling in this test case? It has a function definition that conflicts.